Source/Scene/QuadtreePrimitive.js

```diff
@@ -249,7 +249,7 @@ define([
             this._tileProvider.beginUpdate(context, frameState, commandList);
 
             selectTilesForRendering(this, context, frameState);
-            processTileLoadQueue(this, context, frameState);
+            processTileLoadQueue(this, context, frameState, commandList);
```

Comment:
I think you are starting to see what I mean that passing this separate parameters around is a pain, and we might be better off putting everything in `frameState`, which is still fits well conceptually.